### PR TITLE
Feature/order update

### PR DESCRIPTION
**Pull request by programerscoolguys**

### CODE REVIEW

File: cartridges/int_cloudshelf/cartridge/models/cloudshelf/cloudshelfOrderModel.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+'use strict';
+var Logger = require("dw/system/Logger");
+
+/**
+ * @constructor
+ * @classdesc The ClaudShelf order model
```
Comment (ashleyww93) [inline cartridges/int_cloudshelf/cartridge/models/cloudshelf/cloudshelfOrderModel.js:6]: Cloudshelf

File: cartridges/int_cloudshelf/cartridge/models/cloudshelf/cloudshelfOrderModel.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+'use strict';
+var Logger = require("dw/system/Logger");
+
+/**
+ * @constructor
+ * @classdesc The ClaudShelf order model
+ * @param orderObject - a order objects
+ * @param status - ClaudShelf order status
```
Comment (ashleyww93) [inline cartridges/int_cloudshelf/cartridge/models/cloudshelf/cloudshelfOrderModel.js:8]: Cloudshelf

File: cartridges/int_cloudshelf/cartridge/models/cloudshelf/cloudshelfOrderModel.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+/**
+ * @constructor
+ * @classdesc The ClaudShelf order model
+ * @param orderObject - a order objects
+ * @param status - ClaudShelf order status
+ */
+function orderModel(orderObject, status) {
+    if (orderObject) {
+        try {
+            var claudShelfOrderId =  JSON.parse(orderObject.custom.cloudshelfData).cloudshelfBasketId;
```
Comment (ashleyww93) [inline cartridges/int_cloudshelf/cartridge/models/cloudshelf/cloudshelfOrderModel.js:13]: Cloudshelf

File: cartridges/int_cloudshelf/cartridge/models/cloudshelf/cloudshelfOrderModel.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+        } catch (err) {
+            Logger.warn(
+                'cloudshelfOrderModel error durig parsing cloudshelfData: {0}',
+                JSON.stringify(err)
+            );
+        }
+        this.id = claudShelfOrderId;
```
Comment (ashleyww93) [inline cartridges/int_cloudshelf/cartridge/models/cloudshelf/cloudshelfOrderModel.js:20]: Cloudshelf

**Review by ashleyww93 — CHANGES_REQUESTED** [review]
Small typos, but looks good to me (although I don't understand SFCC)

File: cartridges/int_cloudshelf/cartridge/models/cloudshelf/cloudshelfOrderModel.js (hunk 4 above)
Comment (programerscoolguys) [reply cartridges/int_cloudshelf/cartridge/models/cloudshelf/cloudshelfOrderModel.js]: Fixed

File: cartridges/int_cloudshelf/cartridge/models/cloudshelf/cloudshelfOrderModel.js (hunk 2 above)
Comment (programerscoolguys) [reply cartridges/int_cloudshelf/cartridge/models/cloudshelf/cloudshelfOrderModel.js]: Fixed

File: cartridges/int_cloudshelf/cartridge/models/cloudshelf/cloudshelfOrderModel.js (hunk 1 above)
Comment (programerscoolguys) [reply cartridges/int_cloudshelf/cartridge/models/cloudshelf/cloudshelfOrderModel.js]: Fixed

**Review by ashleyww93 — APPROVED** [review]

**Pull request merged**